lib/lte_link_control/lte_lc.c

```diff
@@ -831,10 +845,23 @@ int lte_lc_psm_req(bool enable)
 	if (enable) {
 		char psm_req[40];
 
-		snprintf(psm_req, sizeof(psm_req),
+		if (strlen(psm_param_rptau) == 8 &&
+		    strlen(psm_param_rat) == 8) {
+			snprintf(psm_req, sizeof(psm_req),
```

Comment:
Looks like the indentation of the two lines below is a bit off (might be GitHub interface issue)